model/flow_estimation.py

```diff
@@ -76,7 +78,7 @@ def forward(self, x, flow):
         mask = tmp[:, 4:5]
         return flow, mask
     
-class MultiScaleFlow(nn.Module):
+class MultiScaleFlow(nn.Module, PyTorchModelHubMixin, repo_url="https://github.com/MCG-NJU/VFIMamba", pipeline_tag="video-frame-interpolation", license="apache-2.0"):
```

Comment:
```suggestion
class MultiScaleFlow(nn.Module, PyTorchModelHubMixin, repo_url="https://github.com/MCG-NJU/VFIMamba", library_name="vfi-mamba", pipeline_tag="video-frame-interpolation", license="apache-2.0"):
```

This change would add a `library_name` tag in the model card metadata to all uploaded models (even the ones finetuned and uploaded by other users). You will be able to list them using https://huggingface.co/models?other=vfi-mamba to track usage of your library.